src/Splitio/Domain/AdapterType.cs

```diff
@@ -2,6 +2,7 @@
 {
     public enum AdapterType
     {
-        Redis //Redis by default
+        Redis, //Redis by default
+        RedisCluster
```

Comment:
this is not necessary. please remove it.

Reply:
will do